x/leveragelp/types/position.go

```diff
@@ -32,9 +31,6 @@ func (position Position) Validate() error {
 	if position.LeveragedLpAmount.IsNegative() {
 		return errorsmod.Wrap(ErrPositionInvalid, "leveraged lp amount cannot be negative")
 	}
-	if !position.Leverage.GT(sdk.OneDec()) {
```

Comment:
I think it should >= 1 now

Reply:
If we are removing this value from here, please make sure we have a validation for msg.leverage 
msg.leverage should be >= 1

Reply:
what does it mean if a user use 0.5 ? I am not sure about the use case for < 1 leverage